File: pgm/ABC_Java/ABC_100_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yay = "Yay!";
        final String no = ":(";
        final int one = 1;
        final int eight = 8;
        final int zero = 0;
        final int A = Integer.parseInt (arr[zero]);
        final int B = Integer.parseInt (arr[one]);
        System.out.println (A <= eight && B <= eight ? Yay : no);
        return;
    }
}

File: pgm/ABC_R/ABC_100_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Yay <- "Yay!"
no <- ":("
blank <- ""
space <- " "
eight <- 8L
one <- 1L
two <- 2L
zero <- 0L
A <- strsplit(arr[one], split = space)[[one]][one] |>
    as.integer()
B <- strsplit(arr[one], split = space)[[one]][two] |>
    as.integer()
if (A <= eight && B <= eight) {
    Yay |>
        cat("\n", sep = blank)
} else {
    no |>
        cat("\n", sep = blank)
}
q("no")